### PR TITLE
Fix search bar alignment

### PR DESCRIPTION
**Pull request by ReazerDev**
https://github.com/Sparsh1212/gsocanalyzer/issues/3

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -11,3 +11,9 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```
Comment (Sparsh1212) [inline src/index.css:14]: Never make changes in index.css. Already a CSS file is imported for the advanced search component, try to add the CSS in that file only.

File: src/components/AdvancedSearch.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const AdvancedSearch = (props) => {
 
   return (
     <Container textAlign='center'>
-      <form>
+      <form class="search-form">
```
Comment (Sparsh1212) [inline src/components/AdvancedSearch.js:48]: In react, the class names are given by the property `className`, not the `class`. Please rectify this.

**Comment by Sparsh1212** [comment]
Other than these changes required, the search component is not responsive in the mobile view. Please make it responsive for mobile view also. 
image

You can see the search bar and the filters are aligned inline. Please make the filters align below the search bar in mobile view.

File: src/index.css (hunk 1 above)
Comment (ReazerDev) [reply src/index.css:14]: Oh yeah, I'm blind haha

**Comment by Sparsh1212** [comment]
@ReazerDev Good Job!

**Comment by ReazerDev** [comment]
> @ReazerDev Good Job!

Thank You! :)

**Pull request merged**